tests/unit/RegAddEditOrgPerson.spec.ts

```diff
@@ -363,44 +363,18 @@ describe('Registration Add/Edit Org/Person component', () => {
     wrapper.destroy()
   })
 
-  it('displays form data for proprietor-org (SP) - manual add', async () => {
```

Comment:
remove it because we do not accept manually added SP now

Reply:
As above, if we are only hiding this functionality for now then I think you should disable this unit test instead of deleting it. (Use `it.skip`.)

Reply:
Ditto for other unit test changes below.

Reply:
LGTM